### ISSUE TITLE
Breaks accelerated scrolling

### ISSUE THREAD

**Issue by bessey** [onefinestay/react-lazy-render#2]
(Great library by the way :heart:)

I notice accelerated scrolling comes to an abrupt halt when the user scrolls at speed. I suspect this is because there isn't a large enough buffer of future nodes in the DOM.

**Comment by bessey** [comment]
Perhaps if the number of nodes was configurable this would address the problem.

**Comment by jkimbo** [comment]
Good point! Never came across this problem when testing it but it definitely makes sense to make it configurable.

**Comment by jkimbo** [comment]
`itemPadding` property is now available on v0.2.0

**Comment by bessey** [comment]
Thank you! I've updated but I'm still facing the same problem.. Took some more time to debug this time. 

It seems in Firefox, things are buttery smooth (actually reducing itemPadding to 1 makes things even smoother!). Increasing padding, even to a mammoth amount, has no effect on Chrome or Safari. Within maybe 100ms of me releasing my fingers from the trackpad, the lazy render scrollbar stops hard, and the outer container is pulled down.

Its not a dealbreaker for me anyway, and i must say it feels absolutely fantastic in FF. I wonder why webkit doesnt like it so much... I don't have much experience debugging these things so I'm going to leave it for now

**Comment by jkimbo** [comment]
Hmm interesting. What operating system are you on btw? Also is the same thing happening on the demo page for you (http://onefinestay.github.io/react-lazy-render/)?

**Comment by bessey** [comment]
OS X, but it doesn't happen with the demo, so I'm gonna assume its my
fault. Will continue to investigate

**Comment by jkimbo** [comment]
Ah so are you setting a `key` on your children? I've noticed this problem before where if I set a `key` then it halts the browser scrolling. What I imagine is going on is that since the keys are different React replaces the whole div rather than just replacing the contents of it and that halts scrolling in Chrome. For the demo I don't set any keys since it's not strictly necessary.
